src/components/HeartbeatBar.vue

```diff
@@ -273,31 +284,35 @@ export default {
 }
 
 .hp-bar-big {
-    .beat {
+    .beat-hover-area {
         display: inline-block;
-        background-color: $primary;
-        border-radius: $border-radius;
 
-        &.empty {
-            background-color: aliceblue;
+        &:not(.empty):hover {
+            transition: all ease-in-out 0.15s;
+            opacity: 0.8;
+            transform: scale(var(--hover-scale));
         }
 
-        &.down {
-            background-color: $danger;
-        }
+        .beat {
+            background-color: $primary;
+            border-radius: $border-radius;
+            pointer-events: none;
```

Comment:
Because the tooltip momentarily disappears when crossing between the beat-hover-area and the beat.

Reply:
Could you add this as a comment?
I think this might be usefull for other readers of this code.

```suggestion
            /* Added because the tooltip momentarily disappears when crossing between the beat-hover-area and the beat */
            pointer-events: none;
```